shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -254,10 +253,10 @@ void PlatformViewChannel::HandleMethodCall(
 
         result->Success();
       } else if (method == "setDirection") {
-        FT_LOGD("PlatformViewChannel setDirection");
+        FT_LOGW("PlatformViewChannel setDirection - not implemented");
         result->NotImplemented();
       } else {
-        FT_LOGD("Unimplemented method: %s", method.c_str());
+        FT_LOGW("Unimplemented method: %s", method.c_str());
```

Comment:
Maybe you can replace these with `FT_UNIMPLEMENTED()` and also update the definition of `FT_UNIMPLEMENTED()`, from `FT_LOGD` to `FT_LOGI` or `FT_LOGW`.

Reply:
I am not sure if FT_UNIMPLEMENTED() will be suitable here. Here the logs says that method.c_str() is not implemented. By using the FT_UNIMPLEMENTED(), log will suggest that 'PlatformChannel::HandleMethodCall()' is not implemented, by printing something like this:
W/ConsoleMessage(20835): platform_channel.cc: HandleMethodCall(259) > UNIMPLEMENTED!

IMHO, current log is more suitable in this context. What do you think about my argumentation?

Besides of above discussion, I will fix the definition of FT_UNIMPLEMENTED to use LOGW instead of LOGD. Thanks.

Reply:
I'm not actually sure what the purpose of the `FT_UNIMPLEMENTED()` macro is (when do we need it?), but your comment sounds reasonable. Thanks for your careful consideration.

Reply:
I think that the  concept was to use it inside a stub method as:

void Foo::Bar() {
  FT_UNIMPLEMENTED();
}
in this conditions, log contains valid name of not implemented function.

Then I will leave above log 'as is'.